### PR TITLE
Use raw status and translate to text here

### PR DESCRIPTION
**Pull request by astrandb**
New version of pyeasee library and change in manifest.json needed to make this fly.
Have not yet changed to @Hellowlol status descriptions since I have not been able to verify them.

### CODE REVIEW

File: custom_components/easee/const.py [hunk 1]
```diff
@@ -460,3 +457,67 @@
         "icon": None,
     },
 }
+
+EQ_ONLINE = "eq_online"
```
Comment (fondberg) [inline custom_components/easee/const.py:461]: Isnt it better to use online and offline strings?

File: custom_components/easee/controller.py [hunk 2]
```diff
@@ -256,8 +258,11 @@ async def refresh_sites_state(self, now=None):
             charger_id = charger_data.charger.id
             site_state = sites_state[charger_data.site.id]
 
-            charger_data.state = site_state.get_charger_state(charger_id)
-            charger_data.config = site_state.get_charger_config(charger_id)
+            charger_data.state = site_state.get_charger_state(charger_id, True)
```
Comment (fondberg) [inline custom_components/easee/controller.py:261]: Would be nice to use named parameters

File: custom_components/easee/controller.py [hunk 3]
```diff
@@ -267,9 +272,9 @@ async def refresh_equalizers_state(self, now=None):
         for equalizer_data in self.equalizers_data:
             equalizer_data.state = await equalizer_data.equalizer.get_state()
             if equalizer_data.state["isOnline"]:
-                equalizer_data.state["isOnline"] = "ONLINE"
+                equalizer_data.state["isOnline"] = "eq_online"
```
Comment (fondberg) [inline custom_components/easee/controller.py:275]: Const exists

File: custom_components/easee/entity.py [hunk 4]
```diff
@@ -30,22 +31,34 @@ def round_to_dec(value, decimals=None, unit=None):
...
+
 def round_1_dec(value, unit=None):
     return round_to_dec(value, 1, unit)
 
+
 def round_0_dec(value, unit=None):
     return round_to_dec(value, None, unit)
 
+
+def map_ea_status(value, unit=None):
```
Comment (fondberg) [inline custom_components/easee/entity.py:47]: map_charger_status is probably a better name

**Review by fondberg — CHANGES_REQUESTED** [review]

File: custom_components/easee/controller.py (hunk 2 above)
Comment (astrandb) [reply custom_components/easee/controller.py:261]: OK

File: custom_components/easee/controller.py (hunk 3 above)
Comment (astrandb) [reply custom_components/easee/controller.py:275]: Result of a string replace that ran away...

File: custom_components/easee/entity.py (hunk 4 above)
Comment (astrandb) [reply custom_components/easee/entity.py:47]: OK

**Comment by astrandb** [comment]
I moved to a new development pc tonight and a few settings files sneaked onboard in this PR. I can try to remove them - tomorrow morning.

**Comment by astrandb** [comment]
Works fine with new pyeasee library.

**Pull request merged**